hangman.rb

```diff
@@ -0,0 +1,125 @@
+
+require 'colorize'
+
+class Hangman
+
+ 
```

Comment:
I would avoid having additional whitespace that you don't need - you'd want to reduce this to just one line of whitespace